### PR TITLE
feat(chat): emoji with chat

### PR DESCRIPTION
**Pull request by fob413**
#### What does this PR do?
- Users can chat with emojis
#### Description of Task to be completed?
- As a user in a game I want to be able to use emojis in the group chat so that I have a more enjoyable and expressive chat experience
#### How should this be manually tested?
- Start a game and chat
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/150398719

### CODE REVIEW

**Comment by fob413** [comment]
@andela-mpitan please merge so Troy can continue his work

File: public/views/chat.html [hunk 1]
```diff
@@ -8,8 +8,26 @@
...
+<script>
+  $(document).ready(function () {
+    $("#emojionearea1").emojioneArea({
+      pickerPosition: "top",
+      tonesStyle: "bullet",
+      events: {
+        keyup: function (editor, event) {
+          if (event.which == 13) {
+            angular.element('#app-container').scope().sendMessage(this.getText());
+            // $("#messages").animate({ scrollTop: $('#messages').prop("scrollHeight")}, 1000);
```
Comment (VanPaitin) [inline public/views/chat.html]: please remove this ghost code if we are not going to be using it

File: public/views/chat.html [hunk 2]
```diff
@@ -8,8 +8,26 @@
...
+            this.setText('');
+          }
+        }
+    	}
+    });
+  })
+</script>
```
Comment (VanPaitin) [inline public/views/chat.html]: You might want to insert a black line at the end of the file

**Review by VanPaitin — CHANGES_REQUESTED** [review]
Looks good. Just minor checks and then I'll merge

**Pull request merged**